Slider/Slider.js

```diff
@@ -106,6 +107,14 @@ const SliderBase = (props) => {
 		onWheel: handle(
 			forProp('disabled', false),
 			forwardWithPrevent('onWheel'),
+			(ev, {wheelEventInterval}) => {
```

Comment:
If the outside is a scroller, preventDefault is required to prevent scrolling.
So this code should be moved on handleIncrementByWheel and handleDecrementByWheel respectively.
(maybe

const handleDecrementByWheel = handle(
	isActive,
	isDecrementByWheel,
	preventDefault,
	stop,
	checkInterval, //maybe here
	isNotMin,
	emitChange(-1)
);

Reply:
It's applied.